stylegan_code_finder/lightning_modules/base_lightning.py

```diff
@@ -21,6 +23,10 @@ def __init__(self, configs: dict):
         self.segmentation_plotter = SegmentationPlotter(configs)
         self.num_val_visualization = configs['num_val_visualization']
         self.save_hyperparameters()
+        self.precision_metric = Precision(configs['num_classes'])
+        self.recall_metric = Recall(configs['num_classes'])
+        self.h_precision = HandwritingPrecision(configs['num_classes'])

```

Comment:
more descirptive naming `handwriting_precision` is not super long but more readable